packages/modelviewer.dev/examples/postprocessing/index.html

```diff
@@ -0,0 +1,306 @@
+<!--
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>&lt;model-viewer&gt; PostProcessing Examples</title>
+  <meta charset="utf-8">
+  <meta name="description" content="&lt;model-viewer-effects&gt; post processing examples">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link type="text/css" href="../../styles/examples.css" rel="stylesheet" />
+  <link type="text/css" href="../../styles/docs.css" rel="stylesheet" />
+  <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+
+  <!-- Import Three.js using an import-map -->
+  <script type="importmap">
+    {
+      "imports": {
+        "three": "https://cdn.jsdelivr.net/npm/three@0.150.1/build/three.module.js"
+      }
+    }
+  </script>
+
+  
+  <!-- Import the no-three component -->
+  <script type="module" src="../../node_modules/@google/model-viewer/dist/model-viewer-no-three.js"></script>
```

Comment:
duplicate?

Reply:
Here is where I'm actually bringing the module in, in the example it's just to show how it works. Bringing it in the example seems to cause some bugs sometimes where the postprocessing just doesn't work, not sure why